projects/yolox-pose/configs/yolox-pose_s_8xb32-300e_coco.py

```diff
@@ -92,7 +92,7 @@
 
 # pipelines
 pre_transform = [
-    dict(type='LoadImageFromFile', file_client_args=_base_.file_client_args),
+    dict(type='mmpose.LoadImage', file_client_args=_base_.file_client_args),
```

Comment:
```suggestion
    dict(type='mmpose.LoadImage', backend_args=_base_.backend_args),
```
Since `file_client_args` is going to be deprecated soon, I suggest using `backend_args`.

Reply:
Meanwhile, please update `backend_args = dict(backend='local')` in `default_runtime.py`